app/policies/movie_policy.rb

```diff
@@ -7,24 +7,8 @@ def show?
     true
   end
 
-  def new?
-    manager?
-  end
-
-  def edit?
-    new?
-  end
-
-  def create?
-    manager?
-  end
-
-  def update?
-    manager?
-  end
-
-  def destroy?
-    update?
+  %i[new? edit? create? update? destroy?].each do |method|
```

Comment:
Feel free to ignore my comment, but I feel in duty to inform you about the potential drawbacks of using metaprogramming techniques in places where they aren't necessary. Personally (you can argue with me :wink:) it decreases the readability of the code, especially for people who are not enough experienced with Ruby.

I encourage you to read [this](https://www.pluralsight.com/blog/software-development/7-deadly-sins-of-ruby-metaprogramming) & [this](https://www.quora.com/Why-don-t-Rails-developers-use-Ruby-metaprogramming).

Reply:
I thought I'm following DRY principle, but I get your point :) I will change it back. Thanks for the reeding recommendation